sharedVariables/src/MinecraftVersion.kt

```diff
@@ -0,0 +1,25 @@
+package at.skyhanni.sharedvariables
+
+import org.gradle.jvm.toolchain.JavaLanguageVersion
+
+enum class MinecraftVersion(
+    val versionName: String,
+    val javaVersion: Int,
+) {
+    MC189("1.8.9", 8),
+    MC11200("1.12", 8),
+    MC11202("1.12.2", 8),
+    MC1144("1.14.4", 8),
+    MC11605("1.16.5", 8),
+    MC121("1.21", 21),
+    ;
```

Comment:
is having the enum delimiter `;` be dangling convention /genq

Reply:
does skyhanni have a standard formatting for kotlin; if not might be better discussion in an issue/new pr

Reply:
Skyhanni has a lint setting that your ide can use. especially here, we explicitly want to show trialing commas always.

Reply:
> is having the enum delimiter ; be dangling convention /genq

i dont know if that is a convention for skyhanni or kotlin in general, but i prefer having all my enums declared with a comma and a semicolon on the next line so i can comment / uncomment entries with a line comment without breaking anything. same goes for argument lists